inputFiles/poromechanicsFractures/poromechanicsFractures.ats

```diff
@@ -82,7 +82,14 @@ decks = [
              restart_step=0,
              check_step=12,
              restartcheck_params=RestartcheckParameters(atol=1e-05,
-                                                        rtol=4e-04))
+                                                        rtol=4e-04)),
+   TestDeck(
+        name="PoroElastic_dfm_PEBICrack_smoke",
+        description='Plane strain poro-elastic problems with conforming vertical fracture (unstructured PEBI grid)',
+        partitions=((1, 1, 1), (3, 1, 1)),
+        restart_step=0,
+        check_step=10,
```

Comment:
I may be wrong, but my understanding is that with `restart_step=0` are we rerunning the simulation twice. In other words, we are not doing any restart check. This comment applies also to other tests not introduced/modified in this PR.